src/components/ProjectsSection/index.js

```diff
@@ -163,6 +174,15 @@ function ProjectsSection() {
     // code to run on component mount
     handleSubmit();
   }, []);
+
+  if (error) {
+    return (
+      <Container>
+        <Error>{error}</Error>
+      </Container>
+    );
+  }
```

Comment:
Not really happy with this being here, but didn't have time to refactor it. I'd rather see that the week-selector would stay and only the loader is switched out for the error message, but since the `data` is used below this, we must cut off the processing here and render the error.